ZCAnimatedLabel/AppDelegate.h

```diff
@@ -2,16 +2,13 @@
 //  AppDelegate.h
 //  ZCAnimatedLabel
 //
-//  Created by Chen Zhang on 2/13/15.
-//  Copyright (c) 2015 somewhere. All rights reserved.
+//  Created by Glenn Posadas on 12/10/20.
```

Comment:
Why do you do this?

Reply:
I apologize. It was intentional. lol. I will never ever steal someone else's work. It's probably due to my copy-pasting.

Reply:
Will push a new commit in a bit. I remember now, that one was auto-generated, as we all know.